lighthouse-core/gather/gatherers/dobetterweb/domstats.js

```diff
@@ -39,9 +39,11 @@ function createSelectorsLabel(element) {
   if (idAttr) {
     name += `#${idAttr}`;
   }
-  const className = element.className;
+  // svg elements return SVGAnimatedString for .className, which is an object.
+  // Use the class attr instead.
+  const className = element.getAttribute && element.getAttribute('class');
```

Comment:
I think elem.classList toString()s to the same values.  Can't double-check this ATM tho.